libcxx/test/libcxx/transitive_includes/cxx17.csv

```diff
@@ -640,6 +640,7 @@ experimental/utility compare
 experimental/utility cstddef
 experimental/utility cstdint
 experimental/utility cstdlib
+experimental/utility cstring
```

Comment:
That this adds new dependencies is a bit unexpected to me. I thought that we already had `std::hash` in `<utility>`, but that we didn't makes me think we shouldn't back-port this after all.

Reply:
Yes i think it's not that problem to support from Cxx17

Reply:
> I thought that we already had `std::hash` in `<utility>`, but that we didn't makes me think we shouldn't back-port this after all.

Oh... libc++'s `<utility>` used to provide `std::hash`, but such extension (?) was removed by 69d5a6662115499198ebfa07a081e98a6ce4b915 (https://reviews.llvm.org/D104002).

Reply:
Let's only add it in C++26 for now. It's much easier to add features than to remove them.